### PR TITLE
[netcore] Add xunit summary script

### PR DESCRIPTION
**Pull request by EgorBo**
```
make xunit-summary
```
it prints some compact xunit report - a similar python script is used by coreclr (or corefx):
```
=========================================================================================================
Common.Tests.dll                                              Total:1144    Failed:0       Time:9.2 sec
CoreFx.Private.TestUtilities.Tests.dll                        Total:87      Failed:0       Time:0.2 sec
Invariant.Tests.dll                                           Total:296     Failed:0       Time:0.4 sec
Microsoft.Bcl.AsyncInterfaces.Tests.dll                       Total:10      Failed:0       Time:0.2 sec
Microsoft.CSharp.Tests.dll                                    Total:4796    Failed:0       Time:2.6 sec
Microsoft.VisualBasic.Core.Tests.dll                          Total:21748   Failed:0       Time:3.2 sec
Microsoft.Win32.Primitives.Tests.dll                          Total:1       Failed:0       Time:0.2 sec
Microsoft.XmlSerializer.Generator.Tests.dll                   Total:83      Failed:0       Time:0.7 sec
System.AppContext.Tests.dll                                   Total:15      Failed:0       Time:0.1 sec
System.Buffers.Tests.dll                                      Total:78      Failed:0       Time:1.9 sec
....
System.Xml.XmlSchema.XmlSchemaValidatorApi.Tests.dll          Total:362     Failed:0       Time:0.6 sec
System.Xml.XmlSchemaSet.Tests.dll                             Total:482     Failed:0       Time:0.9 sec
System.Xml.XmlSerializer.ReflectionOnly.Tests.dll             Total:228     Failed:0       Time:1.7 sec
System.Xml.XmlSerializer.Tests.dll                            Total:230     Failed:0       Time:1.8 sec
System.Xml.Xsl.XslCompiledTransformApi.Tests.dll              Total:1621    Failed:0       Time:7.8 sec
System.Xml.Xsl.XslTransformApi.Tests.dll                      Total:1593    Failed:0       Time:4.7 sec
XsltCompiler.Tests.dll                                        Total:1       Failed:0       Time:0.1 sec
=========================================================================================================

Total test suits:     205
Total tests run:      387451
Total tests passed:   384547
Total tests failed:   108
Total tests skipped:  2796
Total duration:       15 min
```

### CODE REVIEW

File: netcore/Makefile [hunk 1]
```diff
@@ -157,7 +158,7 @@ run-tests-corefx-%: prepare update-tests-corefx
 	cp corefx/restore/corefx-restore.runtimeconfig.dev.json corefx/tests/extracted/$*/xunit.console.runtimeconfig.dev.json
 	cp corefx/restore/corefx-restore.runtimeconfig.dev.json corefx/tests/extracted/$*/Microsoft.DotNet.RemoteExecutorHost.runtimeconfig.dev.json
 	cp corefx/restore/corefx-restore.dll corefx/tests/extracted/$*/corefx-restore.dll
-	sed -i -e 's/9.9.9/$(NETCOREAPP_VERSION)/g' corefx/tests/extracted/$*/*.runtimeconfig.json
+	sed -i -e 's/3.0.0\"/$(NETCOREAPP_VERSION)\"/g' corefx/tests/extracted/$*/*.runtimeconfig.json
```
Comment (EgorBo) [inline netcore/Makefile]: corefx zips now use `"version": "3.0.0"` instead of 9.9.9 in json files

**Comment by EgorBo** [comment]
hm.. I suspect some test failures need fresh commits from coreclr/corefx (the latest mirror happened 4 days ago)

**Comment by steveisok** [comment]
@monojenkins build failed

**Comment by EgorBo** [comment]
I'll leave only the python script in this PR and move other bits (Array related + version updates) to a separate one

**Pull request merged**